src/explore-education-statistics-common/src/components/TabsSection.tsx

```diff
@@ -56,14 +59,16 @@ const TabsSection = forwardRef<HTMLElement, TabsSectionProps>(
 
     return hasRendered ? (
       <section
-        aria-labelledby={onMedia(tabProps['aria-labelledby'])}
+        aria-labelledby={
+          !isSingleTab ? onMedia(tabProps['aria-labelledby']) : ''
```

Comment:
Would suggesting making the single tab branch use `undefined` to prevent the `aria-labelledby` attribute appearing in the markup as it would just be empty.